TensorFI/fiConfig.py

```diff
@@ -239,6 +239,80 @@ def __init__(self,fiParams):
 			self.skipCount = 0
 	# End of constructor
 
+	def updateInstance(self, op, confFile):
+		global fileName
+
+		#if op.value in Ops:
+		instances = confFile['Instances']
+
+		if instances is None:
+			instances = []
+		
+		found = False
+		for value in confFile['Ops']:
+			if (op.value in value) or ('ALL' in value):
+				found = True
+				break
```

Comment:
A `break` doesn't work on an if. You get the error `'break' outside loop`. Please test your code before you submit.

Reply:
I'm not quite sure why this error is happening. The break is inside the for loop on line 252 and it works on my end

Reply:
It's highly unlikely GitHub inserts extra tabs in your code. Can you correct, test and send a final version of this? I don't want to be stuck in this review/debugging loop.